Makefile.main

```diff
@@ -169,6 +169,14 @@ test-coverage:
 	fi; \
 	bash <(curl -s https://codecov.io/bash) $${args}; \
 
+validate-commit: assets
```

Comment:
is calling to "assets" what I was saying is that this shoudn't be in the common CI file, since is not a general stuff. and its calling to a rule that doesn't exists

Reply:
yes. the idea project can use anything for "assets". And this command just checks that nothing has changed, no matter how you generated your assets. Please close the PR if you think it's not generic enough!  👍

Reply:
but you don't need to add assets here, can be added to the concrete Makefile like I do here:
https://github.com/mcuadros/engine/blob/playground/playground/Makefile#

So lets add the rule without the "assets" depedency but, also will require to include a proper explanation of how to use this rule.

Reply:
thank you for suggestions! Dependency has been removed, I renamed it to `no-changes-in-commit` and added documentation.